config/schemas/config.yaml

```diff
@@ -65,40 +71,110 @@ $defs:
         - boolean
     properties:
       chunkLimitSize:
+        title: Chunk limit size for Fluentd buffer
```

Comment:
I've not added titles to simple types, do they get rendered? I haven't looked that close. :sweat_smile:

Reply:
Have to dubble check this :slightly_smiling_face:

Reply:
It renders by the type and inside the type. 
![image](https://github.com/elastisys/compliantkubernetes-apps/assets/12862587/d40621e5-82ce-4f5c-b9d4-86363638dc58)
![image](https://github.com/elastisys/compliantkubernetes-apps/assets/12862587/9d9175df-e6b4-4899-8121-00d80d7d07f6)

Reply:
Then I should add for the ones I've been working on as well.